app/src/main/java/unipd/se18/ocrcamera/inci/Inci.java

```diff
@@ -0,0 +1,81 @@
+package unipd.se18.ocrcamera.inci;
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+
+import com.opencsv.CSVReader;
+
+/**
+ * Class for Inci database management
+ * The database is in CSV format, using openCSV to parse the file.
+ * For more informations about openCSV:
+ * http://opencsv.sourceforge.net/
+ * @author Francesco Pham
+ */
+public class Inci {
+
+    private static final String TAG = "Inci";
+
+    private static final int COL_COSING_REF_NO = 0;
+    private static final int COL_INCI_NAME = 1;
+    private static final int COL_DESCRIPTION = 6;
+    private static final int COL_FUNCTION = 8;
+
+    /**
+     * Load inci database and return list of Ingredient objects
+     * @param inciDbStream InputStream from inci db csv file
+     * @author Francesco Pham
+     */
+    public static ArrayList<Ingredient> getListIngredients(InputStream inciDbStream){
+        Reader reader = new BufferedReader(new InputStreamReader(inciDbStream));
+
+        ArrayList<Ingredient> listIngredients = new ArrayList<>(); //initializing list of ingredients
+
+        //initialize openCSV reader
+        CSVReader csvReader = new CSVReader(reader);
+
+        //skip first line containing field names
+        try {
+            csvReader.skip(1);
+        } catch (IOException e) {
+            Log.e(TAG, "Error skipping first line");
+        }
```

Comment:
I think it is better to separate the try-catch block because even if the reader could not skip the first line, it is preferable to  try continue anyway with reading the CSV file. If you put everything in a try-catch block and the reader throws an exception at `csvReader.skip(1);` then it doesn't even try to read the CSV making all the app completely unfunctional.